Assets/Tests/InputSystem/Plugins/UITests.cs

```diff
@@ -4016,6 +4016,260 @@ public IEnumerator UI_DisplayIndexMatchesDisplayMultiplePointers()
+        button.OnPointerClick(new PointerEventData(EventSystem.current) { button = PointerEventData.InputButton.Left });
+        //Debug.Log(called);
+        Assert.AreEqual(called, true);
+    }
+
+    void ChangeCanvasRenderMode(Canvas canvas, RenderMode mode)
+    {
+        canvas.renderMode = mode;
+    }
+
+    [Test]
+    public void ButtonClickTest()
+    {
+        var allParent = GameObject.Instantiate(Resources.Load("AllParent"));
+
+        var go = GameObject.Find("Left Eye Canvas Screen Camera").transform.Find("Left Button");
+        Button leftButton = go.GetComponent<Button>();
+        Assert.IsNotNull(leftButton, "Can not find Left Button!");
+
+        Button rightButton = GameObject.Find("Right Eye Canvas Screen Camera").transform.Find("Right Button").GetComponent<Button>();
+        Assert.IsNotNull(leftButton, "Can not find Right Button!");
+
+        Canvas leftEyeCanvas = GameObject.Find("Left Eye Canvas Screen Overlay").GetComponent<Canvas>();
+        Assert.IsNotNull(leftEyeCanvas, "Can not find Left Eye Canvas Screen Overlay!");
+
+        Canvas rightEyeCanvas = GameObject.Find("Right Eye Canvas Screen Overlay").GetComponent<Canvas>();
+        Assert.IsNotNull(rightEyeCanvas, "Can not find Right Eye Canvas Screen Overlay!");
+
+        //ClickBothButtonOnCanvasOverlayMode
+        ChangeCanvasRenderMode(leftEyeCanvas, RenderMode.ScreenSpaceOverlay);
+        ChangeCanvasRenderMode(rightEyeCanvas, RenderMode.ScreenSpaceOverlay);
+        ClickButton(leftButton);
+        ClickButton(rightButton);
+
+        //ClickBothButtonOnCanvasCameraMode
+        ChangeCanvasRenderMode(leftEyeCanvas, RenderMode.ScreenSpaceCamera);
+        ChangeCanvasRenderMode(rightEyeCanvas, RenderMode.ScreenSpaceCamera);
+        ClickButton(leftButton);
+        ClickButton(rightButton);
+
+        //ClickBothButtonOnCanvasOverlayMode&CameraMode
+        ChangeCanvasRenderMode(leftEyeCanvas, RenderMode.ScreenSpaceOverlay);
+        ChangeCanvasRenderMode(rightEyeCanvas, RenderMode.ScreenSpaceCamera);
+        ClickButton(leftButton);
+        ClickButton(rightButton);
+
+        GameObject.DestroyImmediate(allParent);
+    }
+
+    [UnityTest]
+    public IEnumerator DragImageTest()
+    {
+        var playAllParent = GameObject.Instantiate(Resources.Load("PlayAllParent"));
+
+        //var m_Canvas = new GameObject("Canvas").AddComponent<Canvas>();
+        //m_Canvas.renderMode = RenderMode.ScreenSpaceOverlay;
+        //m_Canvas.gameObject.AddComponent<GraphicRaycaster>();
+        var m_Canvas = GameObject.Find("Left Eye Canvas Screen Overlay").GetComponent<Canvas>();
+        m_Canvas.targetDisplay = 0; //set 1 on secondary screen
+
+        var m_Image = new GameObject("Image").AddComponent<Image>();
+        m_Image.gameObject.transform.SetParent(m_Canvas.transform);
+        RectTransform imageRectTransform = m_Image.GetComponent<RectTransform>();
+        imageRectTransform.sizeDelta = new Vector2(100f, 100f);
```

Comment:
If you take a look at `CreateTestUI()` you can see the kind of scene they setup for you.

They already have some stuff you can click/drag on so you won't need to add images.